ab7/api-boilerplate
# [ISSUE-4] Resolve: ImportError: attempted relative import with no known parent package

### Description

This PR resolves the following error when trying to run alembic migrations:

```
ImportError: attempted relative import with no known parent package
```

The `data` directory was consolidated into the `api` directory which made import resolution much simpler.

### Issue

https://github.com/ab7/api-boilerplate/issues/4

### Testing

* Pull branch and up the application from scratch (`docker-compose down -v` if already running + delete migrations).
* Verify api and database logs are clean via README instructions.
* Verify that `http://localhost:8000/` returns hello world response.
* Create and run database migrations via README instructions.
* Verify GET and POST requests work calling the `http://localhost:8000/users` endpoint.
